relay/paseo/src/genesis_config_presets.rs

```diff
@@ -126,7 +125,7 @@ fn default_parachains_host_configuration() -> HostConfiguration<polkadot_primiti
 		minimum_backing_votes: 1,
 		node_features: NodeFeatures::EMPTY,
 		async_backing_params: AsyncBackingParams {
-			max_candidate_depth: 3,
+			max_candidate_depth: 2,
```

Comment:
Also this value has changed

Reply:
Let's stick with the ones we (and Polkadot) are running with as we only have local configs in the genesis presets :+1: 

```
  asyncBackingParams: {
    maxCandidateDepth: 3
    allowedAncestryLen: 2
  }
 ```